src/server/ghService/helpers/packageImageHelper.ts

```diff
@@ -0,0 +1,36 @@
+import { isNullOrUndefined } from "util";
```

Comment:
just a thing i do; usually group imports into 3 - 1 for node stuff, 1 for libs, and 1 for project files in that order

Reply:
done